test/test_ops.py

```diff
@@ -339,11 +349,106 @@ def test_out(self, device, dtype, op):
         sample = samples[0]
         # call it normally to get the expected result
         expected = op(*sample.input, *sample.args, **sample.kwargs)
-        # call it with out=... and check we get the expected result
-        out_kwargs = sample.kwargs.copy()
-        out_kwargs['out'] = out = torch.empty_like(expected)
-        op(*sample.input, *sample.args, **out_kwargs)
-        self.assertEqual(expected, out)
+
+        def _test(tested_op):
+            # call it with out=... and check we get the expected result
+            out_kwargs = sample.kwargs.copy()
+            out_kwargs['out'] = out = torch.empty_like(expected)
+            tested_op(*sample.input, *sample.args, **out_kwargs)
+            self.assertEqual(expected, out)
+
+        _test(op)
+        for a_op in op.aliases:
+            _test(a_op)
+
+    @ops([op for op in op_db if op.aliases])
+    def test_jit_alias_remapping(self, device, dtype, op):
```

Comment:
@eellison would you take a look at this port of the test from test_op_aliases.py? @vfdev-5 is creating a version of that test that uses OpInfos instead of the custom AliasInfos that test_op_aliases.py uses. The idea is to eventually remove the AliasInfos by adding their metadata to OpInfos.